mix.exs

```diff
@@ -21,7 +21,7 @@ defmodule GpxEx.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:sweet_xml, "~> 0.6.0"}
+      {:sweet_xml, "~> 0.7.0"}
```

Comment:
For this library's purposes, I _think_ 0.6.6 that we were using previously is actually compatible with 0.7.3, so we could potentially specify a broader supported range here, albeit at the cost of additional testing. 0.6.x is incompatible with a lot of libraries in the wild, including `ExAws`.